paper/paper.md

```diff
@@ -99,15 +99,15 @@ As a cellular automata, Rescal-snow cells also have discrete states. Our sinteri
 
 ### Good practices in computational snow science
 We aim to demonstrate good practices that will encourage robust, reproducible science by releasing Rescal-snow through Journal of Open Source Science.
-Our work is aimed at gemorphologists and snow scientists, and we use this model frequently while working with students.
+Our work is aimed at geomorphologists and snow scientists, and we use this model frequently while working with students.
```

Comment:
This may be the only "real" typo. I don't recall if JOSS wants American or British English. If British English is acceptable, then ignore/reject the other changes.